probe-rs/src/memory/mod.rs

```diff
@@ -122,6 +135,12 @@ pub trait MemoryInterface {
     /// Returns [`Error::MemoryNotAligned`] if this does not hold true.
     fn write_word_32(&mut self, address: u64, data: u32) -> Result<(), Error>;
 
+    /// Write a 16bit word at `address`.
+    ///
+    /// The address where the write should be performed at has to be word aligned.
```

Comment:
What is meant with `word aligned` here? Aligned to 16 bits?

Reply:
Clarified docs, all the places that said "word aligned" now say "multiple of N".